src/commands/address-decode.cpp

```diff
@@ -25,6 +25,9 @@
 #include <bitcoin/explorer/prop_tree.hpp>
 #include <bitcoin/explorer/config/wrapper.hpp>
 
+namespace libbitcoin {
+namespace explorer {
+
 using namespace bc;
 using namespace bc::explorer;
```

Comment:
All of these should be removed from inside of the `libbitcoin::explorer` namespce:

```cpp
using namespace bc;
using namespace bc::explorer;
```